test/karma.conf.js

```diff
@@ -30,14 +20,26 @@ module.exports = function(config) {
       terminal: false
     },
     customLaunchers: {
-      ChromeHeadlessNoSandbox: {
+      ChromeHeadlessDisableAutoplayPolicy: {
         base: 'ChromeHeadless',
-        flags: ['--no-sandbox', '--autoplay-policy=no-user-gesture-required']
-      },
-      ChromeCanaryHeadlessNoSandbox: {
-        base: 'ChromeCanaryHeadless',
-        flags: ['--no-sandbox', '--autoplay-policy=no-user-gesture-required'],
-       }
+        flags: ['--no-sandbox', '--no-user-gesture-required']
```

Comment:
there's a `--mute-audio` flag which might be nice here. less annoying bip bops :)